src/server_manager/model/digitalocean.ts

```diff
@@ -44,13 +44,18 @@ export enum Status {
   MISSING_BILLING_INFORMATION,
 }
 
+export interface AccountInfo {
+  status: Status;
+  warning?: string;
+}
+
 export interface Account {
   // Gets a globally unique identifier for this Account.
   getId(): string;
   // Returns a user-friendly name (email address) associated with the account.
   getName(): Promise<string>;
   // Returns the status of the account.
-  getStatus(): Promise<Status>;
+  getStatus(): Promise<AccountInfo>;
```

Comment:
This is called getStatus, but it doesn't return a status.
Perhaps we should make `Status` be a richer object with methods like `needsEmailVerification()`, `warning()`, `isLocked()`, `hasReachedLimit()`

No need for the enum equality checks

Reply:
Done (using `readonly` fields.  I did not include `hasReachedLimit`, because this is not relevant at login time.